machiavelli-cards/src/main/java/pl/jellytech/machiavelli/cards/utils/TimeUtils.java

```diff
@@ -0,0 +1,23 @@
+package pl.jellytech.machiavelli.cards.utils;
+
+import java.util.concurrent.TimeUnit;
+
+public class TimeUtils {
+    private static final int MILLIS_PER_SECOND = 1000;
+
+    public static long MinutesToMilis(int minutes) {
+        return TimeUnit.MINUTES.toMillis(minutes);
+    }
+
+    public static long TicksToMinutes(long ticks) {
+        return MilisecondsToMinutes(TicksToMilliseconds(ticks));
+    }
+
+    public static long MilisecondsToMinutes(long milis) {
+        return TimeUnit.MILLISECONDS.toMinutes(milis);
+    }
+
+    public static int TicksToMilliseconds(long ticks) {
+        return (int) (ticks % MILLIS_PER_SECOND);
```

Comment:
perhaps switch to Integer.parse instead of casting directly - it's a bit safer
+ if you're gonna update please change the method names to lower letter

Reply:
Integer.parseInt dosen't take long value as parameter. Long wrapper can return int value